app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -19,6 +21,9 @@ case class Project(
   description: String,
   visibility: Visibility,
   tileVisibility: Visibility,
+  isAOIProject: Boolean,
+  aoiCadence: Long, /* Milliseconds */
```

Comment:
You and @echeipesh were against that initially, as you wanted the ability to define multiple AOIs for a single Project and have them checked at the same time.